config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
The last dicussion on Friday was essentially how pod discovery is only for "weird stuff" and that it doesn't allow setting straightforward job labels.
So I'm confused what we want now. Seems not possible to give a generic pod discovery respecting the job label.